libcontainerd/remote/client.go

```diff
@@ -568,8 +568,9 @@ func (c *client) CreateCheckpoint(ctx context.Context, containerID, checkpointDi
 
 	var cpDesc *v1.Descriptor
 	for _, m := range index.Manifests {
-		if m.MediaType == images.MediaTypeContainerd1Checkpoint {
-			cpDesc = &m
+		m2 := m
```

Comment:
nit: `m := m` would work just as well here and wouldn't require a variable called `m2`

Reply:
Agreed. Don't know why I didn't do so 👍  let me update